website/content/gateway/getting_started/rsa_signature/rsa_signature.md

```diff
@@ -0,0 +1,35 @@
+---
+title: "Rsa_signature"
+date: 2022-04-12T14:46:29+02:00
+anchor: "rsa-signature"
+weight: 40
+---
+#### RSA signature
```

Comment:
`####` renders as an `<h4>`, but there's no CSS font set for `<h4>`s, so the headers end up being rendered with the default font rather than `"Circular", sans-serif` as the other headers.

In the old docs at https://docs.gateway.clearhaus.com/#rsa-signature, the "RSA Signature" header is also an `<h3>`, so perhaps this was also meant to be an `<h3>`? I don't see a reason for it to be an `<h4>`.

I see a total of 34 `####` or `#####`s. They'll presumably all have to be changed.

I do also find it a little curious that there's no default font style for the whole page, but rather repeated numerous times. There's 16 occurrences of `font-family: "Calibre", sans-serif;` and 26 occurrences of `font-family: "Circular", sans-serif;` in the current CSS, yet no styles applied globally to `body` or a selector like `h1,h2,h3,h4,h5,h6`.